apps/wallet/src/ui/app/pages/layout/Layout.module.scss

```diff
@@ -7,7 +7,9 @@
     align-items: center;
     justify-content: center;
     flex-flow: column nowrap;
+    background: url('_assets/images/welcome_bg.svg') no-repeat center center;
```

Comment:
Should we include the background conditionally? Currently it's always there and because the view is limited in popup size the content above covers the background.

Reply:
applying this to just `dynamic-size`

Reply:
done